crates/compiler/parse/src/header.rs

```diff
@@ -23,6 +23,16 @@ impl<'a> HeaderType<'a> {
             HeaderType::Platform { .. } | HeaderType::Package { .. } => &[],
         }
     }
+    pub fn to_string(&'a self) -> &str {
```

Comment:
🎨 No need to change, but this would usually be called `as_str` in Rust, and the argument can just be `&self` (doesn't need to say `&'a self`)